### PR TITLE
internationalization -- use "Playlist" instead of "Collection"

### PR DESCRIPTION
**Pull request by markwkidd**
This PR begins the process of internationalizing the preference of "Playlist" and removal of "Collection", following my previous PR which made the changes just to the English strings.

I've submitted this as its own first PR because it actually refactors 'COLLECTION_LIST' enums into the existing 'PLAYLISTS_TAB' enums rather than just renaming an enum. I'll continue on from after this PR to make other adjustments to the enums and translations to reflect where we are with English now.

## Unreviewed
* Greek
* Korean
* Polish

## Reviewed
* Chinese (Simplified): asakous
* Chinese (Traditional): asakous
* Dutch: Maxane the Loud
* French: Tatsuya79
* German: m4xw
* Italian: Ryunam
* Portuguese (Brazil): altiereslima and meleu
* Portuguese (Portugal): altiereslima and meleu
* Russian: ofry
* Spanish: alfrix

## Background and methodology

I am an experienced technical writer. Although I do usually describe my work as a writer and communicator as translation, almost always I am translating between professional disciplines, cultures, and dialects rather than foreign languages.

With that being said, I strive to be accurate and complete in my technical work and I do have experience working with languages and content that I don't know first hand. I have a basic grasp of spoken Japanese and can take some responsibility for that translation myself (perhaps along with bparker's help).

I have a certain amount of academic background in German and French, as well as some very light exposure to Spanish and Portuguese. My German vocabulary helps me feel a little bit confident with Dutch but as with all of these languages it would be great to have any help! 😸 

## Most of what we need already exists in every translation

The key thing to know about this PR is that every one of the languages already either had the term "Playlist" translated or is using the English strings for every case. In many cases the translations of Playlist were available in a couple of different sentences, making it even easier to draft a grammatically-appropriate replacement.

This is the starting point for my work to do what I can to make sure that these changes are not simply copy-and-paste jobs. I'm doing what I can for every language to make sure that conjugations, articles, and single/plurals agree. When I have doubts I consult the history and blame of the translation as well as web resources.

## Checking this draft - moving forward

I am sure there are at least a few mistakes in my work. **My intention is for this PR to touch every use of "collection" in the UI strings for every language**.

Once I have finished going through all strings in all languages I will help tag people who have previously translated these files and ask for help from folks I  know who speak these languages.

By making sure that the PR touches all instances of Collection and its translations in the UI strings that were part of the English PR, this will allow as much time and diligence as necessary to go into making sure the translations are correct.

### CODE REVIEW

**Comment by orbea** [comment]
Just curious, how are you getting these translated?

**Comment by markwkidd** [comment]
Thank you for asking @orbea -- I have just updated the OP with information about how I have created this PR.

I have now done all the work I can. Do you speak any languages other than English? If so could you look at these changes and see if things look OK?

**Comment by markwkidd** [comment]
@DEX357 I see that you have been working with the Polish translation. Would you be willing to look at this update to the Polish strings to help make sure I have done things correctly?

**Comment by markwkidd** [comment]
@hug0b and @Tatsuya79 I see that you have worked on the French translation in the past. Would you be willing to check my work here?

**Comment by markwkidd** [comment]
@lotharsm as a recent major contributor to the German translation, are you available to take a look here?

**Comment by markwkidd** [comment]
@hyarsan I see you have contributed to the Dutch translation. Can you take a look at this?

**Comment by markwkidd** [comment]
@m4xw do you have a moment to check the German UI strings being updated here?

**Comment by Tatsuya79** [comment]
French looks fine.

File: intl/msg_hash_de.h [hunk 1]
```diff
@@ -2974,7 +2970,7 @@ MSG_HASH(MENU_ENUM_SUBLABEL_SHOW_ADVANCED_SETTINGS,
 MSG_HASH(MENU_ENUM_SUBLABEL_THREADED_DATA_RUNLOOP_ENABLE,
       "Führe Aufgaben im Hintergrund aus.")
 MSG_HASH(MENU_ENUM_SUBLABEL_PLAYLIST_ENTRY_REMOVE,
-      "Erlaube dem Benutzer, Einträge aus der Sammlung zu entfernen.")
+      "Erlaube dem Benutzer, Einträge aus Wiedergabelisten zu entfernen.")
```
[the remark this answers is not in the document]
Comment (markwkidd) [reply intl/msg_hash_de.h]: changed, thanks!

File: intl/msg_hash_de.c [hunk 2]
```diff
@@ -676,25 +676,25 @@ int menu_hash_get_help_de_enum(enum msg_hash_enums msg, char *s, size_t len)
          snprintf(s, len,
                "Um nach Inhalten zu suchen, gehe zu '%s' und\n"
                "wähle '%s' oder %s'.\n"
-               " \n"
+               "\n"
                "Die Dateien werden werden mit einer Datenbank abgeglichen.\n"
-               "Bei einem Treffer wird die Datei zu einer Sammlung\n"
+               "Bei einem Treffer wird die Datei zu eine Wiedergabeliste\n"
```
[the remark this answers is not in the document]
Comment (markwkidd) [reply intl/msg_hash_de.c]: changed, thanks!

File: intl/msg_hash_de.h [hunk 3]
```diff
@@ -3237,7 +3233,7 @@ MSG_HASH(MENU_ENUM_SUBLABEL_XMB_MAIN_MENU_ENABLE_SETTINGS,
 MSG_HASH(MENU_ENUM_SUBLABEL_CONTENT_SHOW_SETTINGS_PASSWORD,
       "Wird vor dem Verstecken der Einstellungen ein Passwort vergeben, wird es später möglich sein, den Reiter wiederherzustellen, in dem die Funktion 'Aktiviere den Reiter 'Einstellungen' ausgewählt und das Passwort dort eingegeben wird.")
 MSG_HASH(MENU_ENUM_SUBLABEL_PLAYLIST_ENTRY_RENAME,
-      "Erlaube dem Benutzer, Einträge in der Sammlung umzubenennen.")
+      "Erlaube dem Benutzer, Einträge in Wiedergabelisten umzubenennen.")
```
[the remark this answers is not in the document]
Comment (markwkidd) [reply intl/msg_hash_de.h]: changed, thanks!

**Comment by markwkidd** [comment]
Thank you @Tatsuya79 and @m4xw.

@m4xw : is there a better term for Playlist in German? Now seems like the time to get it right.

**Comment by markwkidd** [comment]
@dnjstlr555 I see that you have contributed to the Korean translation recently. Do you have a moment to look at these few updates?

**Comment by markwkidd** [comment]
@ayssia I see that you have been contributing to the Simplified Chinese translation. Do you have a moment to take a look at these updates to see if they are correct?

**Comment by orbea** [comment]
> I have now done all the work I can. Do you speak any languages other than English? If so could you look at these changes and see if things look OK?

Sorry, I can't help with translations.

**Comment by markwkidd** [comment]
@lambolighting you seem to be the primary Greek translator of RetroArch. Do you have time to look at the 5-6 Greek updates in this PR to make sure they are correct?

**Comment by markwkidd** [comment]
@alfrix is it correct that you speak Spanish? If so, do you have a moment to look at these few UI string updates that affect the Spanish translation?

**Comment by markwkidd** [comment]
@losernator thank you for your work on the RetroArch Korean translation in the past.

Are you available to look at the 6 string updates to the translation in this PR? If so, here is a direct link to the Korean updates: https://github.com/libretro/RetroArch/pull/8488/files#diff-38f00011166e6db6516d77038bdd57fb

**Comment by m4xw** [comment]
@markwkidd it's just germans use the english term all the time. Its technically correct but no one ever says that.

File: intl/msg_hash_es.h [hunk 4]
```diff
@@ -5580,7 +5576,7 @@ MSG_HASH(
     )
 MSG_HASH(
     MENU_ENUM_SUBLABEL_PLAYLIST_ENTRY_REMOVE,
-    "Permitir al usuario eliminar entradas de las colecciones"
+    "Permitir al usuario eliminar entradas de las listas de reprodução"
```
Comment (alfrix) [inline intl/msg_hash_es.h]: lista de reproducción

File: intl/msg_hash_es.h [hunk 5]
```diff
@@ -5749,7 +5745,7 @@ MSG_HASH(
     )
 MSG_HASH(
     MENU_ENUM_SUBLABEL_PLAYLIST_DIRECTORY,
-    "Guardar colecciones en esta carpeta"
+    "Guardar listas de reprodução en esta carpeta"
```
Comment (alfrix) [inline intl/msg_hash_es.h]: lista de reproducción

File: intl/msg_hash_es.h [hunk 6]
```diff
@@ -6073,7 +6069,7 @@ MSG_HASH(
     )
 MSG_HASH(
     MENU_ENUM_SUBLABEL_PLAYLIST_ENTRY_RENAME,
-    "Permitir al usuario renombrar entradas en colecciones"
+    "Permitir al usuario renombrar entradas en listas de reprodução"
```
Comment (alfrix) [inline intl/msg_hash_es.h]: lista de reproducción

Comment (markwkidd) [reply intl/msg_hash_es.h]: fixed, thank you!

File: intl/msg_hash_es.h (hunk 5 above)
Comment (markwkidd) [reply intl/msg_hash_es.h]: fixed, thank you!

File: intl/msg_hash_es.h (hunk 4 above)
Comment (markwkidd) [reply intl/msg_hash_es.h]: fixed, thank you!

**Comment by markwkidd** [comment]
pushed some Spanish fixes from @alfrix -- thanks again!

**Comment by markwkidd** [comment]
@asakous because of your previous contributions to the Traditional Chinese translation, would you be willing to look at a few updates?

These are the specific updates in question:
* https://github.com/libretro/RetroArch/pull/8488/files#diff-d16176d844059f19aa4ef6048ced5e89
* https://github.com/libretro/RetroArch/pull/8488/files#diff-bc82bbf045ff8b49cc85324cbfc08dcd

Chinese is not much like any languages I am experienced with, so your advice would be appreciated.

**Comment by markwkidd** [comment]
@altiereslima I have seen your work on the Brazilian Portugese translation. This PR includes a couple of updates to that language. Specifically the changes are in these two files:
* https://github.com/libretro/RetroArch/pull/8488/files#diff-fdeb6fd50e09819feba1469afde468c7
* https://github.com/libretro/RetroArch/pull/8488/files#diff-5c2ac3f0398c50efb91f025704921f0e

Do you have a moment to look at the changes to make sure that they are correct?

**Comment by markwkidd** [comment]
@TotalCaesar659 and @btimofeev thank you for working on the Russian translation of RetroArch in the past. I am hoping that one or both of you would be able to look at a few updates to make sure that they are correct.

The Russian changes are all in one place: https://github.com/libretro/RetroArch/pull/8488/files#diff-6dce842d4e5bc32fd965b1a25cfd57dd

**Comment by markwkidd** [comment]
@meleu is it true that you speak Portuguese? I am hoping to find folks who can look at these brief updates to both the Brazil and Portugal variations.

In case you do speak Portuguese and have time, here are the updates:
* Brazil 1: https://github.com/libretro/RetroArch/pull/8488/files#diff-fdeb6fd50e09819feba1469afde468c7
* Brazil 2: https://github.com/libretro/RetroArch/pull/8488/files#diff-5c2ac3f0398c50efb91f025704921f0e
* Portugal: https://github.com/libretro/RetroArch/pull/8488/files#diff-a216f4256710899ac5b0b949e39d7759

**Comment by altiereslima** [comment]
> @altiereslima I have seen your work on the Brazilian Portugese translation. This PR includes a couple of updates to that language. Specifically the changes are in these two files:
> 
> * https://github.com/libretro/RetroArch/pull/8488/files#diff-fdeb6fd50e09819feba1469afde468c7
> * https://github.com/libretro/RetroArch/pull/8488/files#diff-5c2ac3f0398c50efb91f025704921f0e
> 
> Do you have a moment to look at the changes to make sure that they are correct?

is perfect

**Comment by markwkidd** [comment]
> > @altiereslima 
> > Do you have a moment to look at the changes to make sure that they are correct?
> 
> is perfect

Thank you. Have a good day!

**Comment by meleu** [comment]
> @meleu is it true that you speak Portuguese?

yep. And I checked those diffs. Using "Coleções" and "Lista de Reprodução" is perfect! :ok_hand:

**Comment by markwkidd** [comment]
Thank you @meleu!

File: intl/msg_hash_ru.h [hunk 7]
```diff
@@ -2488,8 +2484,8 @@ MSG_HASH(MENU_ENUM_LABEL_VALUE_BATTERY_LEVEL_ENABLE,
       "Показать заряд батареи")
 MSG_HASH(MENU_ENUM_LABEL_VALUE_SELECT_FILE,
       "Выбрать файл")
-MSG_HASH(MENU_ENUM_LABEL_VALUE_SELECT_FROM_COLLECTION,
-      "Выбрать файл с коллекции")
+MSG_HASH(MENU_ENUM_LABEL_VALUE_SELECT_FROM_PLAYLIST,
+      "Выбрать из Плейлист.")
```
Comment (ofry) [inline intl/msg_hash_ru.h]: "Выбрать из плейлиста." (first dislection, accusative grammar case)

File: intl/msg_hash_ru.h [hunk 8]
```diff
@@ -2839,7 +2835,7 @@ MSG_HASH(MENU_ENUM_SUBLABEL_CORE_UPDATER_AUTO_EXTRACT_ARCHIVE,
 MSG_HASH(MENU_ENUM_SUBLABEL_NETPLAY_REFRESH_ROOMS,
       "Сканировать для поиска новых комнат.")
 MSG_HASH(MENU_ENUM_SUBLABEL_DELETE_ENTRY,
-      "Удалить эту запись из коллекции.")
+      "Удалить эту запись из Плейлист.")
```
Comment (ofry) [inline intl/msg_hash_ru.h]: "Удалить эту запись из плейлиста." (first dislection, accusative grammar case)

File: intl/msg_hash_ru.h [hunk 9]
```diff
@@ -2911,7 +2907,7 @@ MSG_HASH(MENU_ENUM_SUBLABEL_SHOW_ADVANCED_SETTINGS,
 MSG_HASH(MENU_ENUM_SUBLABEL_THREADED_DATA_RUNLOOP_ENABLE,
       "Выполнять задачи в отдельном потоке.")
 MSG_HASH(MENU_ENUM_SUBLABEL_PLAYLIST_ENTRY_REMOVE,
-      "Разрешить пользователю удалять отдельные записи из коллекции.")
+      "Разрешить пользователю удалять отдельные записи из Плейлисты.")
```
Comment (ofry) [inline intl/msg_hash_ru.h]: "Разрешить пользователю удалять отдельные записи из плейлистов." (plural, first dislection, accusative grammar case)

File: intl/msg_hash_ru.h (hunk 7 above)
Comment (markwkidd) [reply intl/msg_hash_ru.h]: Fixed, thank you!

File: intl/msg_hash_ru.h (hunk 8 above)
Comment (markwkidd) [reply intl/msg_hash_ru.h]: Fixed, thank you!

File: intl/msg_hash_ru.h (hunk 9 above)
Comment (markwkidd) [reply intl/msg_hash_ru.h]: Fixed, thank you!

**Comment by asakous** [comment]
> @asakous because of your previous contributions to the Traditional Chinese translation, would you be willing to look at a few updates?
> 
> These are the specific updates in question:
> 
> * https://github.com/libretro/RetroArch/pull/8488/files#diff-d16176d844059f19aa4ef6048ced5e89
> * https://github.com/libretro/RetroArch/pull/8488/files#diff-bc82bbf045ff8b49cc85324cbfc08dcd
> 
> Chinese is not much like any languages I am experienced with, so your advice would be appreciated.

PLAYLIST should be translated to 遊戲列表. in Traditional Chinese 
so new PR should change 戲列表 to 遊戲列表

**Comment by markwkidd** [comment]
Thank you @asakous! I have just pushed a commit ensuring that all Traditional Chinese strings use 遊戲列表 for "Playlist(s)"

Are you are also familiar with Simplified Chinese?

In case you are, there are parallel changes in that language file as well I would be grateful for your opinion about that update: https://github.com/libretro/RetroArch/pull/8488/files#diff-36fbc5538ece6eff33f2ee7bc041ee93

**Comment by asakous** [comment]
> Thank you @asakous! I have just pushed a commit ensuring that all Traditional Chinese strings use 遊戲列表 for "Playlist(s)"
> 
> Are you are also familiar with Simplified Chinese?
> 
> In case you are, there are parallel changes in that language file as well I would be grateful for your opinion about that update: https://github.com/libretro/RetroArch/pull/8488/files#diff-36fbc5538ece6eff33f2ee7bc041ee93

replace 戏列表 to 游戏列表.

**Comment by markwkidd** [comment]
> replace 戏列表 to 游戏列表.

Thanks again @asakous -- I have made this edit.

**Comment by markwkidd** [comment]
Per my conversation with bparker and twinaphex I have deleted the Greek, Korean, and Polish strings that we have not been able to get a review on yet. That way those strings can be flagged in a scripted way as needing attention.

**Comment by orbea** [comment]
I asked a relative with a background in Polish to look over the Polish strings and was told they looked grammatically correct. However they weren't very familiar with some of the specific terms as they do not use them.

**Pull request merged**